contracts/interfaces/IVoucher.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+/**
+ * @notice Interface describing Voucher for redeeming game items
+ * 
```

Comment:
Suggestion of Gardens :

1. TestGarden which mocks stuff for unit tests
2. StageGarden which has logs and we can deploy for UAT and have method to reset states
3. Treasury could be only managing a single currency distribution like DUCKIES
4. MultiAssetTreasury can support 3rd party tokens
5. GameV1, GameV2, DifferentGameV1